packages/react-storage/src/components/StorageImage/__tests__/StorageImage.test.tsx

```diff
@@ -170,6 +170,27 @@ describe('StorageImage', () => {
     );
   });
 
+  it('should pass bucket to getUrl when supplied', () => {
```

Comment:
Note: we're not duplicating the testing of the functionality of the API supporting this, so this test is just confirming that the "bucket" property is successfully passed from the component to the API in the expected format.